benchmarks/distributed/rpc/ps/launcher.py

```diff
@@ -0,0 +1,372 @@
+import argparse
+import copy
+import json
+import os
+from pathlib import Path
+
+import torch.distributed.rpc as rpc
+import torch.multiprocessing as mp
+from torch.distributed.rpc import TensorPipeRpcBackendOptions
+from torch.utils.data import DataLoader
+
+from benchmark_class_helper import (get_benchmark_data_map,
+                                    get_benchmark_model_map,
+                                    get_benchmark_ps_map,
+                                    get_benchmark_trainer_map)
+from BenchmarkConfigurations import BenchmarkConfigurations
+from metrics.ProcessedMetricsPrinter import ProcessedMetricsPrinter
+
+USE_CUDA_RPC = "use_cuda_rpc"
+
+
+def get_name(rank, trainer_count, parameter_server_count):
+    if rank < trainer_count:
+        return "trainer{}".format(rank)
```

Comment:
nit: use f-strings instead of .format. Same comment for rest of formatted strings in file

Reply:
okay.